app/services/mailer/transactional_email.rb

```diff
@@ -0,0 +1,76 @@
+require "mandrill"
+
+class TransactionalEmail
+  attr_reader :mandrill, :template_name, :brand, :merge_params, :message
+
+  module Types
+    WELCOME     = 'welcome'
+    PLAN_CHANGE = 'plan-change'
+    CANCEL_PLAN = 'cancel-plan'
+  end
+
+  def initialize(template_name, brand)
+    @mandrill = Mandrill::API.new(ENV["MANDRILL_API_KEY"])
+    @template_name = template_name
+    @brand = brand
+    @to = brand.twitter_admin.email
+    @email_attributs = get_email_attributes
+    @message = get_message
+  end
+
+  def send
+    @mandrill.messages.send_template(@template_name, @message)
+  end
+
+  private
+
+  def get_message
+    {
+      "tags": @template_name,
+      "merge_vars": [{
+        "rcpt": @to,
```

Comment:
With `attr_reader` you don't need to reference the instance variable. This can just be `to`.